spec/support/shared/unit/provider/useradd_based_user_provider.rb

```diff
@@ -190,6 +192,10 @@
   end
 
   describe "when managing a user" do
+    let(:passwd_s_status) do
```

Comment:
Do we need this? Doesn't seem to be used

Reply:
Replaced with double for usermod shell_out spoofing